File: Leetcode/Leet0867.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        arr = list()
        for i in range(len(matrix[0])):
            temp = []
            for j in range(len(matrix)):
                temp.append(matrix[j][i])
            arr.append(temp)
        return arr

File: Leetcode/Leet0867.java
class Solution {
    public int[][] transpose(int[][] matrix) {
        
        int[][] arr = new int[matrix[0].length][matrix.length];
        
        for(int row = 0; row < matrix[0].length ; row++){
            for(int col=0; col < matrix.length; col++){
                arr[row][col] = matrix[col][row];
            }
        }return arr;
    }
}